### DISCUSSION TITLE
[init] Please make sure you are using the server version of the FTB modpack!

### DISCUSSION THREAD

**Discussion by gjssss** [itzg/docker-minecraft-server#1410] [Q&A]
Although I found the same discussion before, it didn't help my problem at all
I‘m not familiar with docker, and when I use command to run my container
```shell
docker run -d -v /usr/local/games:/data -e TYPE=CURSEFORGE -e VERSION=1.16.5 \
    -e CF_SERVER_MOD=Above+and+Beyond-1.3-Server.zip \
    -e CF_BASE_DIR=/data \
    -p 25565:25565 -e EULA=TRUE --name mc itzg/minecraft-server:java8-multiarch
```
It says
```shell
[init] Please make sure you are using the server version of the FTB modpack!
```

**Comment by itzg** [comment]
Fairly soon I am going to overhaul the modpack handling so I'll add this one to my list of validation packs. 

In the meantime, using it as a generic pack helps sometimes.

**Reply by gjssss** [reply]
Thank you！
